### PR TITLE
Suppress hypothesis.HealtCheck.too_slow on Python 3

### PR DESCRIPTION
**Pull request by Hexcles**
This is a workaround for #22758.

https://hypothesis.readthedocs.io/en/latest/healthchecks.html#hypothesis.HealthCheck

### CODE REVIEW

**Review by jgraham — APPROVED** [review]

File: tools/manifest/tests/test_manifest.py [hunk 1]
```diff
@@ -105,7 +106,10 @@ def manifest_tree(draw):
 
     return output
 
+
 @h.given(manifest_tree())
+# FIXME: Workaround for https://github.com/web-platform-tests/wpt/issues/22758
+@h.settings(suppress_health_check=(h.HealthCheck.too_slow,) if platform.system() != "Linux" else ())
```
Comment (Hexcles) [inline tools/manifest/tests/test_manifest.py:112]:
```suggestion
@h.settings(suppress_health_check=(h.HealthCheck.too_slow,) if sys.version_info.major == 3 else ())
```

File: tools/manifest/tests/test_manifest.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import os
+import platform
```
Comment (Hexcles) [inline tools/manifest/tests/test_manifest.py:2]:
```suggestion
import sys
```

File: tools/manifest/tests/test_manifest.py [hunk 3]
```diff
@@ -119,7 +123,10 @@ def test_manifest_to_json(s):
 
     assert loaded.to_json() == json_str
 
+
 @h.given(manifest_tree())
+# FIXME: Workaround for https://github.com/web-platform-tests/wpt/issues/22758
+@h.settings(suppress_health_check=(h.HealthCheck.too_slow,) if platform.system() != "Linux" else ())
```
Comment (Hexcles) [inline tools/manifest/tests/test_manifest.py:129]:
```suggestion
@h.settings(suppress_health_check=(h.HealthCheck.too_slow,) if sys.version_info.major == 3 else ())
```

**Comment by Hexcles** [comment]
And it just happened on Linux, too... Looks like Python 3 on all platforms is affected cc @svillar .

**Comment by Hexcles** [comment]
@jgraham FYI this is now disabled in Python 3 on all platforms.

**Pull request merged**